ext/opcache/zend_file_cache.c

```diff
@@ -1825,7 +1825,9 @@ static void zend_file_cache_unserialize(zend_persistent_script  *script,
 	zend_file_cache_unserialize_early_bindings(script, buf);
 }
 
-zend_persistent_script *zend_file_cache_script_load(zend_file_handle *file_handle)
+static zend_persistent_script file_cache_validate_success_script;
+
+zend_persistent_script *zend_file_cache_script_load(zend_file_handle *file_handle, bool validate_only)
```

Comment:
I would prefer renaming this to `zend_file_cache_script_load_ex()` and keeping the signature for `zend_file_cache_script_load()` the same so we don't need to adjust all callers.

Reply:
@iluuu1994 , sorry for taking so long to get back to you! I've been ill for what feels like forever, finally feeling better and getting back up to speed with things slowly.

How is the most recent change in this regard?

Reply:
No worries. Health comes first. It looks good in this regard!